releasenotes/notes/moved-wazuh-certs-b5173d264f543b14.yaml

```diff
@@ -0,0 +1,16 @@
+---
+upgrade:
+  - |
+    The path used for Wazuh certificates has changed. They are now stored in
+    environment directory e.g
+    ``$KAYOBE_CONFIG_PATH/environments/<environment>/wazuh/certificates``
+    or ``$KAYOBE_CONFIG_PATH/wazuh/certificates`` if not using
+    environments. The certificates should be moved from
+    ``$KAYOBE_CONFIG_PATH/ansible/wazuh/certificates`` to the new
+    location.
+  - |
+    The ``local_custom_certs_path`` variable has been removed. Custom wazuh
+    certificates should be moved to
+    ``$KAYOBE_CONFIG_PATH/environments/<environment>/wazuh/certificates/certs``
+    if using environments, or
+    ``$KAYOBE_CONFIG_PATH/wazuh/certificates/certs`` if not.
```

Comment:
```suggestion
  - |
    The ``local_custom_certs_path`` variable has been removed. Custom wazuh
    certificates should be moved to
    ``$KAYOBE_CONFIG_PATH/environments/<environment>/wazuh/certificates``
    if using environments, or
    ``$KAYOBE_CONFIG_PATH/wazuh/certificates`` if not.
```

Reply:
Are we happy to say to move the contents of the directory?